test/init.vim

```diff
@@ -1,5 +1,5 @@
 set nocompatible
-let &runtimepath = '~/.vim/bundle/wiki.vim,' . &runtimepath
+let &runtimepath = expand('<sfile>:p:h:h') . ',' . &runtimepath
```

Comment:
Even simpler:

```vim
let &runtimepath = '../,' . &runtimepath
```

I.e., no reason not to just use relative paths here, is it?

Reply:
sure, looks good